spec/models/article_destroy_spec.rb

```diff
@@ -22,8 +22,11 @@
     it "queues BustCacheJob with user and organization article_ids" do
       expect do
         article.destroy
-      end.to have_enqueued_job(Articles::BustMultipleCachesJob).exactly(:once).
-        with([user_article.id, org_user_article.id, org_article.id].sort)
+      end.to change(Articles::BustMultipleCachesWorker.jobs, :size).by(1)
+
+      enqueued_job = Articles::BustMultipleCachesWorker.jobs.pop
+
+      expect(enqueued_job["args"].first).to eq([user_article.id, org_user_article.id, org_article.id].sort)
```

Comment:
you can use `sidekiq_assert_enqueued_with` for this, see https://github.com/thepracticaldev/dev.to/issues/5305#issuecomment-570264567